test/tools/Modules/HelpersRemoting/HelpersRemoting.psm1

```diff
@@ -258,3 +259,324 @@ function Get-PipePath {
     }
     "$([System.IO.Path]::GetTempPath())CoreFxPipe_$PipeName"
 }
+
+##
+## SSH Remoting helper functions for writing remoting tests
+##
+
+function Get-WindowsOpenSSHLink
+{
+    # From the Win OpenSSH Wiki page (https://github.com/PowerShell/Win32-OpenSSH/wiki/How-to-retrieve-links-to-latest-packages)
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
Maybe save the current value and reset at the end

Reply:
Will do.